agents/rpc/handler.php

```diff
@@ -3,9 +3,9 @@
 $defaultMode = "list";
 $mode = preg_replace("[^A-Za-z]",'', array_key_exists("mode",$_REQUEST) ? $_REQUEST["mode"] : $defaultMode );
 $table = preg_replace("[^A-Za-z_]",'', array_key_exists("table",$_REQUEST) ? $_REQUEST["table"] : '' );
-include_once($SERVER_ROOT.'/classes/AgentManager.php');
-include_once($SERVER_ROOT.'/classes/RdfUtility.php');
-include_once($SERVER_ROOT.'/classes/UuidFactory.php');
+include_once($SERVER_ROOT . '/classes/AgentManager.php');
+include_once($SERVER_ROOT . '/classes/utilities/RdfUtil.php');
+include_once($SERVER_ROOT . '/classes/utilities/UuidFactory.php');
```

Comment:
I think line 206
`if (UuidFactory::isValid($uuid)) { `
should be changed to
`if (UuidFactory::is_valid($uuid)) { `

?